Enum discriminant changes (#153)
minirust/minirust: spec/lang/types.md

        align: Align,
    },
    Enum {

>         /// The map of discriminants to variants. Each variant is given by a type and its

Reviewer: ```suggestion
        /// The map variants, each identified by a discriminant. Each variant is given by a type and its
```
